csharp/Platform.Unsafe/ByteArrayExtensions.cs

```diff
@@ -8,31 +8,29 @@
 namespace Platform.Unsafe
 {
     /// <summary>
-    /// <para>
-    /// Represents the byte array extensions.
-    /// </para>
-    /// <para></para>
+    /// <para>Represents a set of extension methods for byte arrays.</para>
+    /// <para>Представляет набор методов расширения массивов байт.</para>
     /// </summary>
     public unsafe static class ByteArrayExtensions
     {
         /// <summary>
-        /// <para>
-        /// Returns the structure using the specified bytes.
-        /// </para>
-        /// <para></para>
+        /// <para>Returns <paramref name="structure"/> if the length of the byte array with the specified <paramref name="bytes"/> is equal to the size of the structure.</para>
+        /// <para>Возвращает <paramref name="structure"/>, если длина байтового массива при указанных <paramref name="bytes"/> равна размеру структуры.</para>
         /// </summary>
-        /// <typeparam name="TStruct">
-        /// <para>The struct.</para>
-        /// <para></para>
-        /// </typeparam>
+        /// <typeparam name="TStruct"><para>The element's structure type.</para><para>Тип структуры элемента.</para></typeparam>
         /// <param name="bytes">
-        /// <para>The bytes.</para>
-        /// <para></para>
+        /// <para>An array of bytes that will be converted to <typeparamref name="TStruct"/> type.</para>
+        /// <para>Массив байтов, который будет преобразован в тип <typeparamref name="TStruct"/>.</para>
         /// </param>
-        /// <returns>
-        /// <para>The structure.</para>
-        /// <para></para>
-        /// </returns>
+        /// <exeption cref="ArgumentNotEmpty">
+        /// <para>Thrown when <paramref name="bytes"/> is empty.</para>
+        /// <para>Выбрасывается, когда <paramref name="bytes"/> пустой.</para>
+        /// </exeption>
+        /// <exeption cref="ArgumentMeetsCriteria">
```

Comment:
There is no `ArgumentMeetsCriteria` exception type.
You can look at source code:
https://github.com/linksplatform/Exceptions/blob/6549e7798dd8cba362e175dde324662f889235ba/csharp/Platform.Exceptions/EnsureExtensions.cs#L73